.github/ui-e2e-tests/src/app/visibility/application/service-detail/service-detail.po.ts

```diff
@@ -0,0 +1,8 @@
+import { by } from 'protractor';
```

Comment:
Looks like `backend` entities screens are not covered. But, can be added as follow-up PR.